_docs-sources/intro/environment-setup/recommended_tools.md

```diff
@@ -0,0 +1,84 @@
+# Recommended Tools
+
+* [AWS CLI](https://aws.amazon.com/cli/) - The AWS CLI is a powerful tool that lets perform most of the actions that you can do via the AWS Console.
+* [AWS Vault](https://github.com/99designs/aws-vault#installing) - AWS Vault is a slick tool that allows you to easily use different sets of AWS credentials to run commands (`aws`, `terragrunt`, `packer`, et cetera). Once you've gone beyond a single AWS account, a tool like AWS vault is immensely helpful.
+* [Docker](https://www.docker.com/get-started) - Run applications in a container that contains everything needed to run the application without affecting the host environment.
+* [Golang](https://go.dev/dl/) - Golang is an exciting language that has taken the DevOps / infrastructure world by storm. Terraform, Terragrunt, and Terratest are all written in Go.
+* [Kubectl](https://kubernetes.io/docs/tasks/tools/) - If you need to control your Kubernetes cluster, `kubectl` is at the operational core.
+* [Packer](https://www.packer.io/downloads) - Packer lets you build a variety of images, including AWS AMIs and Docker containers. Those images are defined via code for repeatability.
+* [Terraform](https://www.terraform.io/downloads) - Terraform's declarative nature describes your infrastructure as code. If you're using Gruntwork's products, you're using Terraform.
+* [Terragrunt](https://terragrunt.gruntwork.io/docs/getting-started/install/) - Terragrunt is our layer on top of Terraform to enable a highly DRY code base.
+* [Terratest](https://github.com/gruntwork-io/terratest) - If you want to test your infrastructure, Terratest enables you to write those tests for manual testing or integration with a CI/CD pipeline.
+* [tfenv](https://github.com/tfutils/tfenv#installation) - `tfenv` is a version manager for Terraform, similar to `pyenv` for Python.
+* [tgenv](https://gitlab.com/claudiuskastner/tgenv) - `tgenv` is a version manager for Terragrunt
+## Installation
+
+If you would like to run the tools natively on your computer, the links provided above give installation instructions. If you are a user of [brew](https://brew.sh/), then in many cases, a `brew install <TOOL>` will work.
+
+## Tools via Docker
+
+You can use a docker container with all of the tooling built in. If you would prefer to not install all of the tooling locally, then this route is for you.
+
+Save following to a `Dockerfile`:
+
+```dockerfile
```

Comment:
You can add a filename here using the title property: https://docusaurus.io/docs/markdown-features/code-blocks#code-title

Reply:
Addressed in d484bd3